### ISSUE TITLE
Can't See my Courses

### ISSUE THREAD

**Issue by hammanukem** [heliomarpm/udemy-downloader-gui#100]
### Udeler | Udemy Course Downloader

1.11.10

### Running as Business Account

- [ ] Yes

### Steps to reproduce

![Captura de tela de 2023-03-09 09-56-34](https://user-images.githubusercontent.com/12187624/224030117-ca4cf52c-2d2c-4150-be44-5ad6e5ad593b.png)

When I login, I can't see any course. I have a "individual subscription", so I did not buy any of the courses, since this subscription give me access to several courses, but any of the courses that I'm enrroled is visible when I access the app.

### ✔️ Expected Behavior

Be able to see and download the courses that I'm enrroled using the individual subscripton.

### ❌ Actual Behavior / Steps to Reproduce

Just login with a account with individual subscription and try to find the courses that you has enrrol.

**Comment by fu8765** [comment]
Can confirm this is an issue with the business account, no courses visible, not even the ones I'm enrolled to

**Comment by heliomarpm** [comment]
> Can confirm this is an issue with the business account, no courses visible, not even the ones I'm enrolled to, no results when searching either.

sorry I still don't have enough information to verify your problem, I myself have a personal and business account and both manage to list the courses I bought.  the premise of the tool is to list only what was purchased on the Udemy website.  Have you already confirmed that your user can watch the course on the Udemy platform?  Also make sure you are logged in with the same user in the tool

**Comment by heliomarpm** [comment]
Can you send a print accessing the course on Udemy?

**Comment by fu8765** [comment]
> 

yes, my user can watch any of the courses listed on udemy, they just don't show up in the downloader somehow.

**Comment by heliomarpm** [comment]
> > 
> 
> yes, my user can watch any of the courses listed on udemy company portal
> https://mycompany.udemy.com/home/my-courses/learning shows my enrolled courses
> 
> They just don't show up in the downloader, even though I logged in using the token method, with the same screen as above
> Searching courses by name or by course links has "No courses found"

Sorry, I can't find a problem with any of my accounts.  I would need an access token from your company, to try to understand what's going on.

**Comment by heliomarpm** [comment]
Added functionality.
https://github.com/heliomarpm/udemy-downloader-gui/releases/latest
